internal/controller/clusterinstance_controller.go

```diff
@@ -147,9 +158,94 @@ func (r *ClusterInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return ctrl.Result{}, conditions.PatchCIStatus(ctx, r.Client, clusterInstance, patch)
 	}
 
+	// Apply the ACM disaster recovery backup labels to the install template ConfigMaps
+	if err := r.applyACMBackupLabelToInstallTemplates(ctx, log, clusterInstance); err != nil {
+		return requeueWithError(err)
+	}
+
 	return doNotRequeue(), nil
 }
 
+func (r *ClusterInstanceReconciler) applyACMBackupLabelToInstallTemplates(
+	ctx context.Context,
+	log *zap.Logger,
+	clusterInstance *v1alpha1.ClusterInstance,
+) error {
+
+	log = log.Named("applyACMBackupLabelToInstallTemplates")
+
+	// Only apply the ACM disaster recovery backup label to install templates other than those provided
+	// by the SiteConfig Operator
+	siteConfigNS := os.Getenv("POD_NAMESPACE")
+	if siteConfigNS == "" {
+		log.Warn("Could not determine the SiteConfig Operator Namespace")
+	}
+
+	defaultInstallTemplates := []string{
+		ai_templates.ClusterLevelInstallTemplates, ai_templates.NodeLevelInstallTemplates,
+		ibi_templates.ClusterLevelInstallTemplates, ibi_templates.NodeLevelInstallTemplates,
+	}
+
+	applyDRLabelFn := func(ref v1alpha1.TemplateRef) error {
```

Comment:
I don't see why this needed to be separate, why not just embed this into `applyDRLabelToTemplatesFn`?

Reply:
I did it intentionally to avoid having to `log.Error` multiple times in the `applyDRLabelFn` (including the final return `r.Patch`).

Reply:
That's fair